<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.340">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>mbzuai-hackathon-2023 - 14-human_genetics</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js" integrity="sha512-c3Nl8+7g4LMSTdrm621y7kf9v3SDPnhxLNhcjFJbKECVnmZHTdo+IRO05sNLTH/D3vA6u1X32ehoLC7WFVdheg==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>
<script type="application/javascript">define('jquery', [],function() {return window.jQuery;})</script>


<link rel="stylesheet" href="../styles.css">
<meta property="og:title" content="mbzuai-hackathon-2023 - 14-human_genetics">
<meta property="og:description" content="">
<meta property="og:site-name" content="mbzuai-hackathon-2023">
</head>

<body class="nav-sidebar docked nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="../index.html">
    <span class="navbar-title">mbzuai-hackathon-2023</span>
    </a>
  </div>
        <div class="quarto-navbar-tools ms-auto">
</div>
          <div id="quarto-search" class="" title="Search"></div>
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
      <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../datasets.html">Datasets</a></li><li class="breadcrumb-item"><a href="../datasets/14-human_genetics.html">14-human_genetics</a></li></ol></nav>
      <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
      </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation docked overflow-auto">
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">HPP Hackathon</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../schedule.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Schedule</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../challenges.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Competition Challenges</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../submission_instructions.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Submission instructions</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../leaderboard.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Leaderboard</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../platform_tutorial.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Getting Started</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../faq.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">FAQ</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../pheno_utils.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">pheno-utils</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true">
 <span class="menu-text">Datasets</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Available datasets</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/1-population.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">1-population</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/2-anthropometrics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">2-anthropometrics</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/3-fundus.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">3-fundus</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/4-liver_ultrasound.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">4-liver_ultrasound</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/5-diet_logging.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">5-diet_logging</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/6-hand_grip.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">6-hand_grip</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/7-blood_pressure.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">7-blood_pressure</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/8-serum_lipidomics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">8-serum_lipidomics</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/9-sleep.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">9-sleep</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/10-vascular.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">10-vascular</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/11-body_composition.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">11-body_composition</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/12-carotid_ultrasound.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">12-carotid_ultrasound</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/13-gut_microbiome.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">13-gut_microbiome</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/14-human_genetics.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text">14-human_genetics</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/15-ecg.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">15-ecg</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/16-blood_tests.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">16-blood_tests</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/17-cgm.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">17-cgm</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/21-medical_conditions.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">21-medical_conditions</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../datasets/22-bone_density.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">22-bone_density</span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#human-genetics" id="toc-human-genetics" class="nav-link active" data-scroll-target="#human-genetics">14 - human genetics</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">14-human_genetics</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<!-- WARNING: THIS FILE WAS AUTOGENERATED! DO NOT EDIT! -->
<section id="human-genetics" class="level1">
<h1>14 - human genetics</h1>
<p>The Human Phenotype Project collects genomic variation data on all its participants. The genomic data together with the Human Phenotype Project deep-phenotypes allows to investigate the progression of disease, and to explore personalized treatments. We genotype millions of positions by low-pass sequencing combined with imputation using gencove platform technologies. Genotype imputation is a process of statistically inferring unobserved genotypes using known haplotypes in a population. The performance of Gencove genotype imputation is very high ( accuracy &gt; 98% ) (Wasik et al.&nbsp;2021).</p>
<p>The information is stored in a number of statistics parquet files: - <code>main.parquet</code>: sample metadata, including QC statistics, paths to PLINK variant files (raw and post-QC), and principal components (PCs). - <code>variant_qc.parquet</code>: variant QC statistics. - <code>relatives/plink_ibs.parquet</code>: IBS calculated by PLINK for pairs of participants. - <code>relatives/king_kinship.parquet</code>: King kinship coefficients for pairs of participants.</p>
<p>And a PLINK text file <code>pca/eigenvec.var</code> containing the principal component loadings.</p>
<div class="cell" data-execution="{&quot;iopub.execute_input&quot;:&quot;2023-04-29T16:15:21.710266Z&quot;,&quot;iopub.status.busy&quot;:&quot;2023-04-29T16:15:21.710087Z&quot;,&quot;iopub.status.idle&quot;:&quot;2023-04-29T16:15:22.584976Z&quot;,&quot;shell.execute_reply&quot;:&quot;2023-04-29T16:15:22.584468Z&quot;}" data-execution_count="2">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> pheno_utils <span class="im">import</span> PhenoLoader</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-execution="{&quot;iopub.execute_input&quot;:&quot;2023-04-29T16:15:22.587076Z&quot;,&quot;iopub.status.busy&quot;:&quot;2023-04-29T16:15:22.586948Z&quot;,&quot;iopub.status.idle&quot;:&quot;2023-04-29T16:15:26.980247Z&quot;,&quot;shell.execute_reply&quot;:&quot;2023-04-29T16:15:26.979800Z&quot;}" data-execution_count="3">
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a>dl <span class="op">=</span> PhenoLoader(<span class="st">'human_genetics'</span>, age_sex_dataset<span class="op">=</span><span class="va">None</span>)</span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a>dl</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display" data-execution_count="3">
<pre><code>DataLoader for human_genetics with
119 fields
1 tables: ['main']</code></pre>
</div>
</div>
<div class="cell" data-execution="{&quot;iopub.execute_input&quot;:&quot;2023-04-29T16:15:26.982087Z&quot;,&quot;iopub.status.busy&quot;:&quot;2023-04-29T16:15:26.981859Z&quot;,&quot;iopub.status.idle&quot;:&quot;2023-04-29T16:15:27.005635Z&quot;,&quot;shell.execute_reply&quot;:&quot;2023-04-29T16:15:27.005217Z&quot;}" data-execution_count="4">
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a>dl.<span class="bu">dict</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display" data-execution_count="4">
<div>


<table class="dataframe table table-sm table-striped small" data-quarto-postprocess="true" data-border="1">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th">field_string</th>
<th data-quarto-table-cell-role="th">description_string</th>
<th data-quarto-table-cell-role="th">parent_dataframe</th>
<th data-quarto-table-cell-role="th">relative_location</th>
<th data-quarto-table-cell-role="th">value_type</th>
<th data-quarto-table-cell-role="th">units</th>
<th data-quarto-table-cell-role="th">sampling_rate</th>
<th data-quarto-table-cell-role="th">item_type</th>
<th data-quarto-table-cell-role="th">array</th>
<th data-quarto-table-cell-role="th">cohorts</th>
<th data-quarto-table-cell-role="th">data_type</th>
<th data-quarto-table-cell-role="th">debut</th>
<th data-quarto-table-cell-role="th">pandas_dtype</th>
</tr>
<tr class="odd">
<th data-quarto-table-cell-role="th">tabular_field_name</th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
<th data-quarto-table-cell-role="th"></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td data-quarto-table-cell-role="th">collection_date</td>
<td>Collection date</td>
<td>The date of downloading Gencove results from t...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Time</td>
<td>Time</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>datetime64[ns]</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">version</td>
<td>Gencove version</td>
<td>Gencove API version 1 or 2</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Categorical (multiple)</td>
<td>NaN</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">genecov_qc_bases</td>
<td>Bases sequenced</td>
<td>number of total bases sequenced. from genecov ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Integer</td>
<td>Count</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">genecov_qc_bases_dedup</td>
<td>Deduplicated bases</td>
<td>number of deduplicated bases. from genecov qc ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Integer</td>
<td>Count</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">gencove_qc_bases_dedup_mapped</td>
<td>Deduplicated bases aligned</td>
<td>number of deduplicated bases that have aligned...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Integer</td>
<td>Count</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">genecov_qc_effective_coverage</td>
<td>Effective coverage</td>
<td>effective coverage. from genecov qc file</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>Precent</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">genecov_qc_fraction_contamination</td>
<td>DNA contamination</td>
<td>contamination by DNA from another sample of th...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>Precent</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">genecov_qc_snps</td>
<td>SNPs covered</td>
<td>number of variants in reference panel that are...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Integer</td>
<td>Count</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">genecov_qc_format_passed</td>
<td>Passed FASTQ format validity</td>
<td>Indicates proper formatting of the input FASTQ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Categorical (single)</td>
<td>Boolean</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">genecov_qc_r1_eq_r2_passed</td>
<td>Passed paired-end count</td>
<td>Indicates number of bases in R1 file equal to ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Categorical (single)</td>
<td>Boolean</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">genecov_qc_r1_r2_ids_match_passed</td>
<td>Passed paired-end match</td>
<td>Indicates R1 read identifiers match R2 read id...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Categorical (single)</td>
<td>Boolean</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">genecov_qc_bases_passed</td>
<td>Passed Bases sequenced</td>
<td>Indicates number of total bases sequenced grea...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Categorical (single)</td>
<td>Boolean</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">genecov_qc_bases_dedup_passed</td>
<td>Passed Deduplicated bases</td>
<td>Indicates number of deduplicated bases greater...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Categorical (single)</td>
<td>Boolean</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">genecov_qc_bases_dedup_mapped_passed</td>
<td>Passed Deduplicated bases aligned</td>
<td>Indicates number of deduplicated bases aligned...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Categorical (single)</td>
<td>Boolean</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">genecov_qc_effective_coverage_passed</td>
<td>Passed Effective coverage</td>
<td>Indicates effective coverage greater than 0. f...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Categorical (single)</td>
<td>Boolean</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">genecov_qc_fraction_contamination_passed</td>
<td>Passed DNA contamination</td>
<td>Indicates contamination by DNA less than 0.06....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Categorical (single)</td>
<td>Boolean</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">genecov_qc_snps_passed</td>
<td>Passed SNPs covered</td>
<td>Indicates number of variants covered by at lea...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Categorical (single)</td>
<td>Boolean</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">heterozygosity_proportion</td>
<td>Heterozygosity proportion</td>
<td>Estimated heterozygosity proportion across a s...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>Precent</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">pass_het_check</td>
<td>Passed Heterozygosity proportion</td>
<td>Indicates heterozygosity proportion is less th...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Categorical (single)</td>
<td>Boolean</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>bool</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">sample_qc_gp_pass_rate</td>
<td>Genotype Probabilities pass rate</td>
<td>Proportion of genotypes with maximum GP greate...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>Precent</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">sample_qc_gp_pass_check</td>
<td>Passed Genotype Probabilities</td>
<td>Indicates proportion of GP greater than 0.9 is...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Categorical (single)</td>
<td>Boolean</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>bool</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">genetic_sex</td>
<td>DNA sex</td>
<td>Sex as determined by the percentage of read-al...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Categorical (single)</td>
<td>Boolean</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">chr_x_proportion</td>
<td>Proportion of alignment to chromosome X</td>
<td>percentage of reads aligned to chromosome X</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>Precent</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">chr_y_proportion</td>
<td>Proportion of alignment to chromosome Y</td>
<td>percentage of reads aligned to chromosome Y</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>Precent</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">sample_qc_gender_match_check</td>
<td>Passed gender check</td>
<td>Indicates inferred_gender and submitted_gender...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Categorical (single)</td>
<td>Boolean</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>bool</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">relatives</td>
<td>Relatives based on IBD estimation</td>
<td>List of second and first degree relatives acco...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>String</td>
<td>Text</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">self_report_ashkenaz_proportion</td>
<td>Self-reported ashkenazi descent proportion</td>
<td>Proportion of great-grandparents with ashkenaz...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>Precent</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">self_report_middleeastern_proportion</td>
<td>Self-reported Middle-Eastern descent proportion</td>
<td>Proportion of great-grandparents with Middle-E...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>Precent</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">self_report_northafrican_proportion</td>
<td>Self-reported North-African descent proportion</td>
<td>Proportion of great-grandparents with ashkenaz...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>Precent</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">self_report_sephardi_proportion</td>
<td>Self-reported Sephardi descent proportion</td>
<td>Proportion of great-grandparents with North-Af...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>Precent</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">self_report_yemen_proportion</td>
<td>Self-reported Yemen descent proportion</td>
<td>Proportion of great-grandparents with Yemen de...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>Precent</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">self_report_unknownother_proportion</td>
<td>Self-reported unknown or other descent proport...</td>
<td>Proportion of great-grandparents with unknown ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>Precent</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">pc1</td>
<td>Principal component 1</td>
<td>Score for projected principal component 1</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>None</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">pc2</td>
<td>Principal component 2</td>
<td>Score for projected principal component 2</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>None</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">pc3</td>
<td>Principal component 3</td>
<td>Score for projected principal component 3</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>None</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">pc4</td>
<td>Principal component 4</td>
<td>Score for projected principal component 4</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>None</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">pc5</td>
<td>Principal component 5</td>
<td>Score for projected principal component 5</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>None</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">pc6</td>
<td>Principal component 6</td>
<td>Score for projected principal component 6</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>None</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">pc7</td>
<td>Principal component 7</td>
<td>Score for projected principal component 7</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>None</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">pc8</td>
<td>Principal component 8</td>
<td>Score for projected principal component 8</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>None</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">pc9</td>
<td>Principal component 9</td>
<td>Score for projected principal component 9</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>None</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">pc10</td>
<td>Principal component 10</td>
<td>Score for projected principal component 10</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>None</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">pc11</td>
<td>Principal component 11</td>
<td>Score for projected principal component 11</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>None</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">pc12</td>
<td>Principal component 12</td>
<td>Score for projected principal component 12</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>None</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">pc13</td>
<td>Principal component 13</td>
<td>Score for projected principal component 13</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>None</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">pc14</td>
<td>Principal component 14</td>
<td>Score for projected principal component 14</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>None</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">pc15</td>
<td>Principal component 15</td>
<td>Score for projected principal component 15</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>None</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">pc16</td>
<td>Principal component 16</td>
<td>Score for projected principal component 16</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>None</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">pc17</td>
<td>Principal component 17</td>
<td>Score for projected principal component 17</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>None</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">pc18</td>
<td>Principal component 18</td>
<td>Score for projected principal component 18</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>None</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">pc19</td>
<td>Principal component 19</td>
<td>Score for projected principal component 19</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>None</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">pc20</td>
<td>Principal component 20</td>
<td>Score for projected principal component 20</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Continuous</td>
<td>None</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>float64</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">used_in_pca_calculation</td>
<td>Included in principal component analysis</td>
<td>Indicates samples was in the input for princip...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>Categorical (single)</td>
<td>Boolean</td>
<td>NaN</td>
<td>Data</td>
<td>Single</td>
<td>10K</td>
<td>Tabular</td>
<td>2019-03-11</td>
<td>bool</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bed_chr_1</td>
<td>PLINK BED file for post QC genotype data chr1</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_fam_chr_1</td>
<td>PLINK FAM file accompanying QC .bed chr1</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bim_chr_1</td>
<td>PLINK BIM file for the post QC genotype data chr1</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bed_chr_2</td>
<td>PLINK BED file for post QC genotype data chr2</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_fam_chr_2</td>
<td>PLINK FAM file accompanying QC .bed chr2</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bim_chr_2</td>
<td>PLINK BIM file for the post QC genotype data chr2</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bed_chr_3</td>
<td>PLINK BED file for post QC genotype data chr3</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_fam_chr_3</td>
<td>PLINK FAM file accompanying QC .bed chr3</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bim_chr_3</td>
<td>PLINK BIM file for the post QC genotype data chr3</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bed_chr_4</td>
<td>PLINK BED file for post QC genotype data chr4</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_fam_chr_4</td>
<td>PLINK FAM file accompanying QC .bed chr4</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bim_chr_4</td>
<td>PLINK BIM file for the post QC genotype data chr4</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bed_chr_5</td>
<td>PLINK BED file for post QC genotype data chr5</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_fam_chr_5</td>
<td>PLINK FAM file accompanying QC .bed chr5</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bim_chr_5</td>
<td>PLINK BIM file for the post QC genotype data chr5</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bed_chr_6</td>
<td>PLINK BED file for post QC genotype data chr6</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_fam_chr_6</td>
<td>PLINK FAM file accompanying QC .bed chr6</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bim_chr_6</td>
<td>PLINK BIM file for the post QC genotype data chr6</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bed_chr_7</td>
<td>PLINK BED file for post QC genotype data chr7</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_fam_chr_7</td>
<td>PLINK FAM file accompanying QC .bed chr7</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bim_chr_7</td>
<td>PLINK BIM file for the post QC genotype data chr7</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bed_chr_8</td>
<td>PLINK BED file for post QC genotype data chr8</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_fam_chr_8</td>
<td>PLINK FAM file accompanying QC .bed chr8</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bim_chr_8</td>
<td>PLINK BIM file for the post QC genotype data chr8</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bed_chr_9</td>
<td>PLINK BED file for post QC genotype data chr9</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_fam_chr_9</td>
<td>PLINK FAM file accompanying QC .bed chr9</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bim_chr_9</td>
<td>PLINK BIM file for the post QC genotype data chr9</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bed_chr_10</td>
<td>PLINK BED file for post QC genotype data chr10</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_fam_chr_10</td>
<td>PLINK FAM file accompanying QC .bed chr10</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bim_chr_10</td>
<td>PLINK BIM file for the post QC genotype data c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bed_chr_11</td>
<td>PLINK BED file for post QC genotype data chr11</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_fam_chr_11</td>
<td>PLINK FAM file accompanying QC .bed chr11</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bim_chr_11</td>
<td>PLINK BIM file for the post QC genotype data c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bed_chr_12</td>
<td>PLINK BED file for post QC genotype data chr12</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_fam_chr_12</td>
<td>PLINK FAM file accompanying QC .bed chr12</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bim_chr_12</td>
<td>PLINK BIM file for the post QC genotype data c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bed_chr_13</td>
<td>PLINK BED file for post QC genotype data chr13</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_fam_chr_13</td>
<td>PLINK FAM file accompanying QC .bed chr13</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bim_chr_13</td>
<td>PLINK BIM file for the post QC genotype data c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bed_chr_14</td>
<td>PLINK BED file for post QC genotype data chr14</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_fam_chr_14</td>
<td>PLINK FAM file accompanying QC .bed chr14</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bim_chr_14</td>
<td>PLINK BIM file for the post QC genotype data c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bed_chr_15</td>
<td>PLINK BED file for post QC genotype data chr15</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_fam_chr_15</td>
<td>PLINK FAM file accompanying QC .bed chr15</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bim_chr_15</td>
<td>PLINK BIM file for the post QC genotype data c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bed_chr_16</td>
<td>PLINK BED file for post QC genotype data chr16</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_fam_chr_16</td>
<td>PLINK FAM file accompanying QC .bed chr16</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bim_chr_16</td>
<td>PLINK BIM file for the post QC genotype data c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bed_chr_17</td>
<td>PLINK BED file for post QC genotype data chr17</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_fam_chr_17</td>
<td>PLINK FAM file accompanying QC .bed chr17</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bim_chr_17</td>
<td>PLINK BIM file for the post QC genotype data c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bed_chr_18</td>
<td>PLINK BED file for post QC genotype data chr18</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_fam_chr_18</td>
<td>PLINK FAM file accompanying QC .bed chr18</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bim_chr_18</td>
<td>PLINK BIM file for the post QC genotype data c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bed_chr_19</td>
<td>PLINK BED file for post QC genotype data chr19</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_fam_chr_19</td>
<td>PLINK FAM file accompanying QC .bed chr19</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bim_chr_19</td>
<td>PLINK BIM file for the post QC genotype data c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bed_chr_20</td>
<td>PLINK BED file for post QC genotype data chr20</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_fam_chr_20</td>
<td>PLINK FAM file accompanying QC .bed chr20</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bim_chr_20</td>
<td>PLINK BIM file for the post QC genotype data c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bed_chr_21</td>
<td>PLINK BED file for post QC genotype data chr21</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_fam_chr_21</td>
<td>PLINK FAM file accompanying QC .bed chr21</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_bim_chr_21</td>
<td>PLINK BIM file for the post QC genotype data c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bed_chr_22</td>
<td>PLINK BED file for post QC genotype data chr22</td>
<td>Raw genotype calls (per chromosome) in binary ...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Compressed binary file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="even">
<td data-quarto-table-cell-role="th">postqc_fam_chr_22</td>
<td>PLINK FAM file accompanying QC .bed chr22</td>
<td>PLINK sample information file accompanying a ....</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
<tr class="odd">
<td data-quarto-table-cell-role="th">postqc_bim_chr_22</td>
<td>PLINK BIM file for the post QC genotype data c...</td>
<td>PLINK variant information file (per chromosome...</td>
<td>NaN</td>
<td>main.parquet</td>
<td>NaN</td>
<td>NaN</td>
<td>NaN</td>
<td>Bulk</td>
<td>Single</td>
<td>10K</td>
<td>Text file</td>
<td>2019-03-11</td>
<td>object</td>
</tr>
</tbody>
</table>

</div>
</div>
</div>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>